config.json

```diff
@@ -14,6 +14,18 @@
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginHash": "sha512",
```

Comment:
Oh, and add in challenge.saltLength to determine how long the challenge's seed/salt should be.